content/docs/2.7/scalers/azure-event-hub.md

```diff
@@ -34,6 +40,9 @@ triggers:
     - `blobMetadata` - For all implementations that store checkpoint information on blob metadata such as current C#, Python, Java and JavaScript Event Hub SDKs.
     - `goSdk` - For all implementations using the [Golang SDK](https://github.com/Azure/azure-event-hubs-go)'s checkpointing, for example Dapr.
     - When no checkpoint strategy is specified, the Event Hub scaler will use backwards compatibility and able to scale older implementations of C#, Python or Java Event Hub SDKs. (see "Legacy checkpointing"). If this behaviour should be used, `blobContainer` is also required.
+- `cloud` - Name of the cloud environment that the Event Hub belongs to. Must be a known Azure cloud environment, or `Private` for Azure Stack Hub or Air Gapped clouds. (valid values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `AzureGermanCloud`, `Private`; default: `AzurePublicCloud`)
```

Comment:
Could you use the convention "name - Description. (Values: `x`, `y`, `z`, Default: `y`, Optional, Extra Info)"
And it'd be awesome if you could update also the old lines (35 and 38) using that pattern